### ISSUE TITLE
Continue Button

### ISSUE THREAD

**Issue by dinero1010** [Shufflepuck/SplashBuddy#89]
Our workflow instructs users to click the Continue button when all items are completed.  We also set all items to 'canContinue' so the expected behavior is that the Continue button is shown instead of the Quit button, regardless of install status for each item.  What would be the most efficient way to set the Continue button label as 'Continue' and keep that static regardless of install status?

**Comment by dwshore** [comment]
Couldn't this be done within the source code? Just replace the text of the button with whatever you'd like and recompile? (NOTE: I don't know if that would break something in the code that's listening for 'quit' specifically. n00b->me)

**Comment by TheBlu** [comment]
@cybertunnel: I have been trying to find a way to change the quit button as well. Do you have any documentation on this? Any help would be appreciated!

**Comment by cybertunnel** [comment]
@TheBlu I don't think we have anything in place in the current released versions. The way that you can change what the button does, and in turn change the button text can be found [here](https://github.com/Shufflepuck/SplashBuddy/wiki/60---Options)
